UefiPayloadPkg/Library/DebugPrintErrorLevelLibHob/DebugPrintErrorLevelLibHob.c

```diff
@@ -13,6 +13,7 @@
 #include <Library/PcdLib.h>
 #include <Library/HobLib.h>
 #include <Guid/DebugPrintErrorLevel.h>
+#include <Guid/UniversalPayloadBase.h>

```

Comment:
Skimming the header file, I don't think it's used here.

Reply:
UNIVERSAL_PAYLOAD_GENERIC_HEADER defined in UniversalPayloadBase.h and used in GetDebugPrintErrorLevel method.

Reply:
UNIVERSAL_PAYLOAD_GENERIC_HEADER is defined in UniversalPayload.h, so we can remove UniversalPayloadBase.h

Reply:
Todo.

Reply:
resolved.